cvat/apps/dataset_manager/project.py

```diff
@@ -156,7 +155,7 @@ def import_dataset(self, dataset_file, importer, **options):
         project_data.soft_attribute_import = True
 
         temp_dir_base = self.db_project.get_tmp_dirname()
-        os.makedirs(temp_dir_base, exist_ok=True)
+
```

Comment:
Why this `makedirs` call was removed? It has to create parent dirs. `TemporaryDirectory` fails, if these dirs do not exist. But what I think is that we don't even need the parent dir here, it's actually better to create a regular temporary directory in `/tmp`.